docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -958,6 +958,15 @@ MINIO_USE_SSL = false
 
 And used by `[attachment]`, `[lfs]` and etc. as `STORAGE_TYPE`.
 
+## MIME type mapping (`download.mimetype.mapping`)
+
+Configuration for set the expected MIME type based on file extensions of downloadable files. Configuration presents in key-value pairs and file extensions starts with leading `.`.
+
+The following configuration set `Content-Type: application/javascript` header when downloading files with `.js` file extension.
```

Comment:
I'm hesitant to have js be the example(same for .ini as well), could you have it be just apk? As JS can be executed by the browesr, so I don't want people copy pasting things without realizing what they are configuring and opening themselves up to security issues.